docs/manual/docs/cmd/todo/list/list-remove.md

```diff
@@ -0,0 +1,37 @@
+# todo list remove
+
+Removes a Microsoft To Do task list
+
+## Usage
+
+```sh
+todo list remove [options]
+```
+
+## Options
+
+Option|Description
+------|-----------
+`--help`|output usage information
+`-n, --name [name]`|The name of the task list to remove. Specify either id or name but not both
+`-i, --id [id]`|The ID of the list to remove. Specify either id or name but not both
```

Comment:
Let's rename this to ``The ID of the task list to remove. Specify either `id` or `name` but not both``